apps/projects/app/components/Panel/NewProject/RepoSelector.js

```diff
@@ -22,20 +22,26 @@ class Repo extends React.Component {
     repoSelected: -1,
   }
 
+  filterAlreadyAdded = repos => {
+    if (this.props.reposAlreadyAdded === undefined) return repos
+    const reposAlreadyAddedIds = this.props.reposAlreadyAdded.map(repo => repo.data._repo)
+    return repos.filter(repo => reposAlreadyAddedIds.indexOf(repo.node.id) === -1)
```

Comment:
IMHO `Array.some()` or `Array.includes()` seem more readable for these cases instead of `Array.indexOf()` and then compare index

Reply:
Improved.